### ISSUE TITLE
Bot crashes when -addparty or -deleteparty is run

### ISSUE THREAD

**Issue by jonasbohmann** [jonasbohmann/democraciv-discord-bot#4]
Whenever I run the -addparty or -deleteparty commands, the bot crashes.

Seems like it's an issue with the asynchronous `await` in config.py. I'll investigate this and leave this issue open to document it.

Traceback:
```
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x758a87b0>
Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x75621880>, 361305.431396705)]']
connector: <aiohttp.connector.TCPConnector object at 0x758a8450>
task was destroyed but it is pending!
task: <Task pending coro=<DiscordWebSocket.close_connection() running at /usr/local/lib/python3.7/site-packages/discord/gateway.py:539> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x75743550>()]> cb=[shield.<locals>._done_callback() at /usr/local/lib/python3.7/asyncio/tasks.py:776]>
Exception ignored in: <coroutine object DiscordWebSocket.close_connection at 0x7573ebb8>
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/discord/gateway.py", line 539, in close_connection
  File "/usr/local/lib/python3.7/site-packages/websockets/protocol.py", line 855, in close_connection
  File "/usr/local/lib/python3.7/site-packages/websockets/protocol.py", line 880, in wait_for_connection_lost
  File "/usr/local/lib/python3.7/asyncio/tasks.py", line 400, in wait_for
  File "/usr/local/lib/python3.7/asyncio/base_events.py", line 657, in call_later
  File "/usr/local/lib/python3.7/asyncio/base_events.py", line 667, in call_at
  File "/usr/local/lib/python3.7/asyncio/base_events.py", line 480, in _check_closed
RuntimeError: Event loop is closed
Task was destroyed but it is pending!
task: <Task pending coro=<Client._run_event() running at /usr/local/lib/python3.7/site-packages/discord/client.py:227> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x75873950>()]>>
Task was destroyed but it is pending!
task: <Task pending coro=<BotBase.close() running at /usr/local/lib/python3.7/site-packages/discord/ext/commands/bot.py:208> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x74d40fb0>()]> cb=[Client._do_cleanup.<locals>.when_future_is_done() at /usr/local/lib/python3.7/site-packages/discord/client.py:498]>
```

**Comment by TheGreatCookieMachine** [comment]
[I dont seem to be having the same issue](https://cdn.discordapp.com/attachments/559880939044470786/563429537183039492/Capture.PNG), however when I searched around about the `Task was destroyed but it is pending!` error, it seems to be caused when the bot takes too long on a function. Sorry I couldnt be of more help.

**Comment by TheGreatCookieMachine** [comment]
After some more testing I still couldnt get the error, however I did find an issue in config.deleteParty() which may have been related to the error when deleting parties (Fixed in Pull Request #5). I am still not sure about the error when creating parties however.
